content/security/advisories/rss.xml.haml

```diff
@@ -0,0 +1,52 @@
+!!! XML
+%rss(version="2.0"
+     xmlns:content="https://purl.org/rss/1.0/modules/content/"
+     xmlns:atom="http://www.w3.org/2005/Atom")
+
+  :ruby
+    require 'asciidoctor'
+    advisories_dir = File.expand_path(File.dirname(__FILE__) + '/../advisory')
+    pages_by_path = site.pages.map { |p| [p.source_path, p] }.to_h
+    adocs = Dir.glob(File.join(advisories_dir, '*.{ad,adoc}'))
+
+    Plugin = Struct.new(:name, :title)
+
+  %channel
+    %title
+      Jenkins Security Advisories
+    %link
+      https://www.jenkins.io/security/advisories/
+    %atom:link(href="#{absolute_link(page.url)}" rel="self" type="application/rss+xml")
+    %description
+      Security advisories published by the Jenkins project
+    %lastBuildDate
+      #{DateTime.now.rfc822}
+    - advisories = Dir.glob(File.join(advisories_dir, '202*.{ad,adoc}'))
```

Comment:
We don't need 2019 and earlier entries, which also makes the below code simpler (advisories before 2018 did not have content metadata).